conda_lock/conda_lock.py

```diff
@@ -389,6 +390,21 @@ def make_lock_files(  # noqa: C901
                 strip_auth=strip_auth,
             )
 
+            if not lock_content:
+                lock_content = new_lock_content
+            else:
+                # Persist packages from original lockfile for platforms not requested for lock
+                packages_not_to_lock = [
+                    dep
+                    for dep in lock_content.package
+                    if dep.platform not in platforms_to_lock
+                ]
+                filtered_lock_content = lock_content.copy(
```

Comment:
Better name: `lock_content_to_persist`?

Reply:
Ya. I'd even rename the original to `original_lock_content`. And maybe `new_lock_content` → `fresh_lock_content` in order to reserve `new_lock_content` for the final result.

I'm a big fan of long and descriptive variable names.

Reply:
>

Ya. I'd even rename the original to original_lock_content. And maybe new_lock_content → fresh_lock_content in order to reserve new_lock_content for the final result.
>
>I'm a big fan of long and descriptive variable names.

:100: same here :)

Reply:
2a1fb78